src/SFA.DAS.ApprenticeCommitments/Data/Models/Apprenticeship.cs

```diff
@@ -85,7 +87,9 @@ private Revision GetRevision(long revisionId)
         }
 
         internal void Confirm(long revisionId, Confirmations confirmations, DateTimeOffset now)
-            => GetRevision(revisionId).Confirm(confirmations, now);
+        {
+            GetRevision(revisionId).Confirm(confirmations, now);
+        }
```

Comment:
Would it be more appropriate to update Apprenticeship ConfirmedOn here?

Reply:
Yes, I think you're right.  That is cleaner 👍